.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+env:
+  - TYPO3_BRANCH=TYPO3_6-2
+  - TYPO3_BRANCH=TYPO3_7-0
```

Comment:
Is it really useful to test against 7.x which is not master? These are meant to be updated as soon as a new 7.x is released.